commons-business/src/main/java/com/appjars/saturn/service/UpdateService.java

```diff
@@ -20,15 +20,22 @@
 package com.appjars.saturn.service;
 
 /**
- * A special kind of service that allows entities update
- * 
- * @author mlopez
+ * Specifies the contract of a generic service that allows updating entities.
  *
- * @param <T>
- * @param <K>
+ * @param <T> type of entity to update
+ *
+ * @author mlopez
  */
 public interface UpdateService<T> {
 
-	void update(T entity);
+  /**
+   * Updates the provided entity.
```

Comment:
I think that we should use the word  `given` instead of `provided`, as `given` was used in the rest of the docs